test/runtime/write.c

```diff
@@ -688,6 +767,8 @@ int main(int argc, char **argv)
         persistence_write_test();
     }
 
+    fs_stress_test();
+
```

Comment:
This should be under the "basic" tests, so this function call should be in the `if (op == WRITE_OP_ALL || op == WRITE_OP_BASIC_ONLY)` block, a few lines above